### ISSUE TITLE
Serious AMP takeover problem - title tags are incorrect

### ISSUE THREAD

**Issue by Frenchomatic** [ahmedkaludi/accelerated-mobile-pages#1955]
It appears that the title tag as defined in the genesis doc title or yoast settings is being ignored and replaced with the page title. This is a big backward step and it needs to be sorted out rapidly. When the site is not in AMP takeover the title shows normally as it has for many months.

**Comment by Frenchomatic** [comment]
It seems like this code from features.php is being used when there is an AMP takeover so most of it fails 

```
if(function_exists('genesis_title')){
			if(is_home() && is_front_page() && !$redux_builder_amp['amp-frontpage-select-option']){
				// Determine the doctitle.
			$genesis_title = genesis_get_seo_option( 'home_doctitle' ) ? genesis_get_seo_option( 'home_doctitle' ) : get_bloginfo( 'name' );

			// Append site description, if necessary.
			$genesis_title = genesis_get_seo_option( 'append_description_home' ) ? $genesis_title . " $sep " . get_bloginfo( 'description' ) : $genesis_title;
			}
			elseif ( is_home() && get_option( 'page_for_posts' ) && get_queried_object_id() ) 
			{ 
				$post_id = get_option( 'page_for_posts' );
				if ( null !== $post_id || is_singular() ) {
					if ( genesis_get_custom_field( '_genesis_title', $post_id ) ) {
						$genesis_title = genesis_get_custom_field( '_genesis_title', $post_id );
					}
				}
			}
			elseif( is_home() && get_option( 'page_on_front' ) && $redux_builder_amp['amp-frontpage-select-option'] ){
				$post_id = get_option('page_on_front');
					if ( null !== $post_id || is_singular() ) {
						if ( genesis_get_custom_field( '_genesis_title', $post_id ) ) {
							$genesis_title = genesis_get_custom_field( '_genesis_title', $post_id );
						}
					}
			}
			else {
				$genesis_title = genesis_default_title( $title );
			}
			if( $genesis_title ){
				$site_title = $genesis_title;
			}
		}
		return esc_html( convert_chars( wptexturize( trim( $site_title ) ) ) );
	}
}

```
The only way I could make things work as a hack was to delete **[return esc_html( convert_chars( wptexturize( trim( $site_title ) ) ) );](url)** . This then forces the use of normal genesis seo titles and put **return esc_html( convert_chars( wptexturize( trim( $genesis_title ) ) ) );**  to get the blog posts page correct here 

```
elseif ( is_home() && get_option( 'page_for_posts' ) && get_queried_object_id() ) 
			{ 
				$post_id = get_option( 'page_for_posts' );
				if ( null !== $post_id || is_singular() ) {
					if ( genesis_get_custom_field( '_genesis_title', $post_id ) ) {
						$genesis_title = genesis_get_custom_field( '_genesis_title', $post_id );
					}
				}
				return esc_html( convert_chars( wptexturize( trim( $genesis_title ) ) ) );
			}
```

Basically, a mess but it seems to work for me.

**Comment by Frenchomatic** [comment]
see #1969

**Comment by Frenchomatic** [comment]
This now looks to be fixed on the current version.
